images/10-qemuguestagent/run

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+for _ in {1..10}
+do
+    if [ -f /run/console-done ];then
+        system-docker exec console sh -c 'echo "docker:tcuser" | chpasswd' > /dev/null 2>&1
```

Comment:
Is it possible that we don't reset the password each time?

Reply:
We can use `cat /etc/shadow | grep docker | awk -F ":" '{print $2}' == "*"` to determine if the password needs to be reset.

Reply:
No. Just make sure `tcuser` password is set once.
You can touch a stamp.
@kingsd041